demo/addons/fileimport/fileimport.html

```diff
@@ -0,0 +1,3 @@
+  <button href="#" id="fileimport-button" title="Importer fichier local" i18n="fileimport.button.title" class="btn btn-default btn-raised">
+   <span class="glyphicon glyphicon-import" aria-hidden="true"></span>
```

Comment:
Don't work in mobile device ?

Reply:
Indeed import does not seem very useful on mobile, so I hided the button for now.

Reply:
I agree.